mcs/tools/mdoc/Test/html.expected/Mono.DocTest.Generic/GenericBase`1+FooEventArgs.html

```diff
@@ -190,34 +190,34 @@
       <a href="../index.html">DocTest</a> : <a href="index.html">Mono.DocTest.Generic Namespace</a></div>
     <div class="SideBar">
       <p>
-        <a href="#T:Mono.DocTest.Generic.GenericBase`1.FooEventArgs">Overview</a>
+        <a href="#T:Mono.DocTest.Generic.GenericBase&lt;U&gt;+FooEventArgs">Overview</a>
       </p>
       <p>
-        <a href="#T:Mono.DocTest.Generic.GenericBase`1.FooEventArgs:Signature">Signature</a>
+        <a href="#T:Mono.DocTest.Generic.GenericBase&lt;U&gt;+FooEventArgs:Signature">Signature</a>
       </p>
       <p>
-        <a href="#T:Mono.DocTest.Generic.GenericBase`1.FooEventArgs:Docs">Remarks</a>
+        <a href="#T:Mono.DocTest.Generic.GenericBase&lt;U&gt;+FooEventArgs:Docs">Remarks</a>
```

Comment:
This is the `transpose` code I was referencing earlier, regarding nested types.

In retrospect, perhaps using `+` is better...